origami.json

```diff
@@ -1,35 +1,38 @@
 {
-    "description": "Accessible, content-aware widget for expanding and collapsing content",
-    "origamiVersion": 1,
-    "origamiType": "module",
-    "browserFeatures": {
-        "required": [
-            "classlist",
-            "customevent"
-        ]
-    },
-    "supportStatus": "active",
-    "support": "https://github.com/Financial-Times/o-expander/issues",
-    "demosDefaults": {
-        "js": "demos/src/demo.js",
-        "sass": "demos/src/demo.scss"
-    },
-    "demos": [
-        {
-            "name": "Simple",
-            "description": "Simple use cases",
-            "template": "demos/src/simple.mustache",
-            "js": "demos/src/demo.js",
-            "expanded": true,
-            "description": "Simple use cases"
-        },
-        {
-            "name": "Variants",
-            "description": "More advanced configurations of the module",
-            "template": "demos/src/variants.mustache",
-            "js": "demos/src/variants.js",
-            "expanded": false,
-            "description": "More advanced configurations of the module"
-        }
-    ]
+	"description": "Accessible, content-aware widget for expanding and collapsing content",
+	"origamiVersion": 1,
+	"origamiType": "module",
+	"browserFeatures": {
+		"required": [
+		"classlist",
+		"customevent"
+		]
+	},
+	"supportStatus": "active",
+	"support": "https://github.com/Financial-Times/o-expander/issues",
+	"demosDefaults": {
+		"js": "demos/src/demo.js",
+		"sass": "demos/src/demo.scss"
+	},
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-expander"
+	},
+	"demos": [
+		{
```

Comment:
Be good to get the indenting consistent. Either drop the `{` back an indent, or indent the contents one more tab.